addons/website_sale/static/src/js/website_sale.js

```diff
@@ -236,6 +236,8 @@ export const WebsiteSale = publicWidget.Widget.extend(VariantMixin, cartHandlerM
                 return;
             }
             if (!data.cart_quantity) {
+                // Ensures last cart removal is recorded
+                sessionStorage.setItem('website_sale_cart_quantity', 0);
```

Comment:
Could you use `browser.sessionStorage` from `@web/core/browser/browser` instead? It use RAM storage in cases where localStorage is not available.
```suggestion
                browser.sessionStorage.setItem('website_sale_cart_quantity', 0);
```

Reply:
Sure :)